src/EVault/modules/Liquidation.sol

```diff
@@ -88,6 +88,8 @@ abstract contract LiquidationModule is ILiquidation, Base, BalanceUtils, Liquidi
         // Violator's health check must not be deferred, meaning no prior operations on violator's account
         // would possibly be forgiven after the enforced collateral transfer to the liquidator
         if (isAccountStatusCheckDeferred(violator)) revert E_ViolatorLiquidityDeferred();
+        // Liquidation can't happen if violator was healthy in this block, in order to prevent flashloan attacks
+        if (getLastAccountStatusCheckTimestamp(violator) == block.timestamp) revert E_SameBlockLiquidation();
```

Comment:
Let's consider making the "min delta" configurable. Currently this is implicitly `1`, but:

* Setting to `0` would allow governors to disable this protection (you could also use this to keep the liquidation tests clean, and avoid a bunch of `skip(1)`s
* Setting to `> 1` would let users choose a minimum expected liveness period, which may vary from chain-to-chain, or other circumstances.